gateway/gateway.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 
 	"github.com/celer-network/goutils/log"
+	sgnbindings "github.com/celer-network/sgn-contract/bindings/go/mainchain"
```

Comment:
how about renaming the package when generate binding in sgn-contract/bindings/go? update this: https://github.com/celer-network/sgn-contract/blob/master/gen_and_push.sh#L56-L57

Reply:
Done